### PR TITLE
Corrected folder, not submodule

### PR DESCRIPTION
**Pull request by hanneshapke**
This PR should fix the problem in github with the listing as a submodule.

### CODE REVIEW

File: gcf/gcf-ai-platform-example/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Google Cloud Function Examples
+
+Example code to infer Google Cloud AI Platform models from Google Cloud Functions
+
+### Example Endpoints
+
+Main.py contains the example code for two endpoints:
```
Comment (gogasca) [inline gcf/gcf-ai-platform-example/README.md]: nit: `main.py`

File: gcf/gcf-ai-platform-example/main.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        request with an argument `sentence` ('application/json')
+    Returns:
+        Response with prediction results
+    """
+    # expected content type is
+    request_json = request.get_json(silent=True)
+    sentence = request_json['sentence']
+
+    service = _connect_service()
+    project = 'drydock'
```
Comment (gogasca) [inline gcf/gcf-ai-platform-example/main.py]: nit: Use generic name for project

File: gcf/gcf-ai-platform-example/main.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
+import json
+
+from flask import current_app as app
+
+import googleapiclient.discovery
+
+
+def _generate_payload(sentence):
```
Comment (gogasca) [inline gcf/gcf-ai-platform-example/main.py]: nit: Add Args and returns. 
Some functions use Function documentation (3.8.3 Functions and Methods)
https://github.com/google/styleguide/blob/gh-pages/pyguide.md

File: gcf/gcf-ai-platform-example/main.py [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    """
+    Helper function to prep the payload data structure. Also a good
+    place to manipulate the payload (e.g., lower-casing if needed)
+    """
+    return {"instances": [{"sentence": sentence}]}
+
+
+def _get_model_meta(service, project, model='demo_model', version=None):
```
Comment (gogasca) [inline gcf/gcf-ai-platform-example/main.py]: nit: Same as above

File: gcf/gcf-ai-platform-example/main.py [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+        meta = response
+    else:
+        response = service.projects().models().get(name=url).execute()
+        meta = response['defaultVersion']
+
+    model_id = meta['name']
+    return meta, model_id
+
+
+def _get_model_prediction(service, project, model='demo_model',
```
Comment (gogasca) [inline gcf/gcf-ai-platform-example/main.py]: nit: Same as above

File: gcf/gcf-ai-platform-example/main.py [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+def get_demo_inference_meta_endpoint():
+    """
+    Endpoint to demonstrate requesting the meta information from Google's AI Platform
+    Args:
+        None
+    Returns:
+        Response with model meta information
+    """
+    service = _connect_service()
+    project = 'drydock'
```
Comment (gogasca) [inline gcf/gcf-ai-platform-example/main.py]: nit: Same as above

File: gcf/gcf-ai-platform-example/main.py [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+    """
+    Endpoint to demonstrate requesting an inference from a model
+    hosted via Google's AI Platform.
+
+    Args:
+        request with an argument `sentence` ('application/json')
+    Returns:
+        Response with prediction results
+    """
+    # expected content type is
```
Comment (gogasca) [inline gcf/gcf-ai-platform-example/main.py]: expected content type is JSON ?

File: gcf/gcf-ai-platform-example/main.py [hunk 8]
```diff
@@ -0,0 +1,97 @@
+import json
```
Comment (gogasca) [inline gcf/gcf-ai-platform-example/main.py]: Add small file documentation.
eg. """Launches a prediction request using AI Platform Prediction API."""

File: gcf/gcf-ai-platform-example/main.py (hunk 3 above)
Comment (hanneshapke) [reply gcf/gcf-ai-platform-example/main.py]: doc string added

File: gcf/gcf-ai-platform-example/main.py (hunk 4 above)
Comment (hanneshapke) [reply gcf/gcf-ai-platform-example/main.py]: doc string added

File: gcf/gcf-ai-platform-example/main.py (hunk 5 above)
Comment (hanneshapke) [reply gcf/gcf-ai-platform-example/main.py]: doc string added

File: gcf/gcf-ai-platform-example/main.py (hunk 6 above)
Comment (hanneshapke) [reply gcf/gcf-ai-platform-example/main.py]: renamed to `demo_project`

File: gcf/gcf-ai-platform-example/main.py (hunk 7 above)
Comment (hanneshapke) [reply gcf/gcf-ai-platform-example/main.py]: docstring amended.

File: gcf/gcf-ai-platform-example/main.py (hunk 8 above)
Comment (hanneshapke) [reply gcf/gcf-ai-platform-example/main.py]: File doc string added with https://github.com/GoogleCloudPlatform/ml-on-gcp/pull/49/commits/f0f0a3fb47bd818027b909c69bfe629990a5b265

File: gcf/gcf-ai-platform-example/main.py (hunk 2 above)
Comment (hanneshapke) [reply gcf/gcf-ai-platform-example/main.py]: Changed to `demo_project`

**Comment by hanneshapke** [comment]
@gogasca Docstrings and renaming of the project id happened in https://github.com/GoogleCloudPlatform/ml-on-gcp/pull/49/commits/f0f0a3fb47bd818027b909c69bfe629990a5b265

Thank you for your review!

**Comment by gogasca** [comment]
LGTM now

**Pull request merged**